src/index.js

```diff
@@ -0,0 +1,141 @@
+"use strict";
+
+const state = {
+  city: 'Portland, OR',
+  temperature: 72,
+  latitude: 45.5152,
+  longitude: 122.6784,
+};
+
+let getCityName;
+let curWeatherEmojis = document.getElementById("curWeatherEmojis");
+let cityDisplayOnLoad = document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${state.city}`;
+
+const updateCityWhileTyping = () => {
+  const input = document.getElementById("defaultCity").value;
+  document.getElementById("now-showing").textContent = 
+  `Now showing the temperature for ${input}`
+  state.city = input;
+};
+
+document.getElementById("sky").addEventListener('change', (event) => {
+  const bgValue = event.target.options[event.target.selectedIndex].value;
+  if( bgValue == 'sunny' ) {
+    document.body.style.backgroundImage = "url(/ada-project-docs/assets/sunshine.png)";      
+    } else if( bgValue == "cloudy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/cloudy.png)";      
+    } else if( bgValue == "rainy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/rain.png)";      
+    } else if( bgValue == "snowy" ){
+      document.body.style.backgroundImage = "url(/ada-project-docs/assets/snow.png)";      
+    }
+});
```

Comment:
Could this have been wrapped inside a function and the function been registered on an event inside `registerEventHandlers` like you do with your other event handler methods?

For example, this could be put in a method called like `updateBackground` and then in registerEventHandlers you could have:
```javascript
const background = document.getElementById("sky");
background.addEventListener('change', updateBackground);
```

Doing so would ensure that your format is uniform throughout the project and makes the code a bit more readable